GNUmakefile.os4

```diff
@@ -240,6 +240,7 @@ ifdef GITTAG
 	$(VERBOSE)sed -i 's/REVISION\t*[[:digit:]]/REVISION\t\t$(MINOR)/g' library/c.lib_rev.h
 	$(VERBOSE)sed -i 's/SUBREVISION\t*[[:digit:]]/SUBREVISION\t\t$(PATCH)/g' library/c.lib_rev.h
 	$(VERBOSE)sed -i 's/clib4.library [0-9]*\.[0-9]*\.[0-9]*/clib4.library $(MAJOR).$(MINOR).$(PATCH)/g' library/c.lib_rev.h
+	$(VERBOSE)sed -i 's/Version: [0-9]*\.[0-9]*\.[0-9]*/Version: $(MAJOR).$(MINOR).$(PATCH)/g' misc/control
```

Comment:
Nitpick alert:
I would update the regexp to `s/Version: [0-9]+\.[0-9]+\.[0-9]+/Version: $(MAJOR).$(MINOR).$(PATCH)/g`
The change is that the regexp now requires at least on digit for MAJOR, MINOR and PATCH.

Reply:
Using `[[:digit:]]\+` should do want you want. It will also match 1+ digits.